packages-native/actions/src/notifications/ScheduleNotification.ts

```diff
@@ -79,12 +82,11 @@ function ScheduleNotification(
         notification.setNotificationId(notificationId);
     }
 
-    return firebase
-        .notifications()
-        .scheduleNotification(notification, {
-            fireDate: date.getTime()
-        })
-        .then(() => true);
+    await firebase.notifications().scheduleNotification(notification, {
```

Comment:
You can return this

Reply:
Is not void, is any

Reply:
https://rnfirebase.io/docs/v5.x.x/notifications/reference/Notifications#scheduleNotification

Reply:
Done